classify.py

```diff
@@ -329,120 +340,118 @@ def evaluate_samesource(desc, dataset, n_frequent_words, tokens_per_sample, prep
     recall = np.round(np.mean(lrs[y_all == 1] > 1), 3)
     precision = np.round(np.mean(y_all[lrs > 1] == 1), 3)
 
-    return cllr, cllrmin, cllrcal, acc, recall, precision
+    return cllr, cllrmin, cllrcal, acc, recall, precision, lrs, y_all
```

Comment:
you may consider using namedtuple:
`Metrics = collections.namedtuple('Metrics', ['cllr', 'cllrmin', 'cllrcal', ...])`

And then: `return Metrics(cllr, cllrmin, ...)`

Reply:
done